kpi/forms.py

```diff
@@ -65,3 +74,31 @@ class Meta:
             # The 'password' field appears without adding it here; adding it
             # anyway results in a duplicate
         ]
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        # Intentional _() call on dynamic string because the default choices
+        # are translated (see static_lists.py)
+        self.fields['sector'].choices = (('', ''),) + tuple(
+            (s, _(s)) for s in constance.config.SECTOR_CHOICES.split('\n')
```

Comment:
I think we should open an issue about replacing `ugettext*` for Django 4. I wouldn't mind moving to `t()` as the alias at the same time. Could you file the issue?

Since you're my reviewer and I know you'll enjoy it, I went ahead and replaced all `_()` with `t()` in this file.